api-spec/WFS3core+STAC.yaml

```diff
@@ -0,0 +1,631 @@
+        default:
+          description: An error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/exception'
+            text/html:
+              schema:
+                type: string
+  '/collections/{collectionId}/items':
+    get:
+      summary: 'retrieve features of feature collection {collectionId}'
+      description: >-
+        Every feature in a dataset belongs to a collection. A dataset may
+        consist of multiple feature collections. A feature collection is often a
+        collection of features of a similar type, based on a common schema.\
+
+        Use content negotiation to request HTML or GeoJSON.
+      operationId: getFeatures
+      tags:
+        - Features
+      parameters:
+        - $ref: '#/components/parameters/collectionId'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/bbox'
+        - $ref: '#/components/parameters/time'
+      responses:
+        '200':
+          description: >-
+            Information about the feature collection plus the first features
+            matching the selection parameters.
+          content:
+            application/geo+json:
+              schema:
+                $ref: '#/components/schemas/featureCollectionGeoJSON'
+            text/html:
+              schema:
+                type: string
+        default:
+          description: An error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/exception'
+            text/html:
+              schema:
+                type: string
+  '/collections/{collectionId}/items/{featureId}':
+    get:
+      summary: retrieve a feature; use content negotiation to request HTML or GeoJSON
+      operationId: getFeature
+      tags:
+        - Features
+      parameters:
+        - $ref: '#/components/parameters/collectionId'
+        - $ref: '#/components/parameters/featureId'
+      responses:
+        '200':
+          description: A feature.
+          content:
+            application/geo+json:
+              schema:
+                $ref: '#/components/schemas/featureGeoJSON'
+            text/html:
+              schema:
+                type: string
+        default:
+          description: An error occured.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/exception'
+            text/html:
+              schema:
+                type: string
+# START STAC-specific operations                
+  /search/stac:
+    post:
+      summary: >-
+        retrieve items matching filters.
+      operationId: searchSTAC
+      tags:
+        - STAC
+      requestBody:
+        content:
+          'application/json':      
+            schema:
+              $ref: '#/components/schemas/searchBody'
+      responses:
+        '200':
+          description: A feature.

```

Comment:
Should add the STAC Item response here, the geojson as an example. The ones in the spec.yaml and wfs-stac yaml are up to date with the latest.